embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/ModuleInitBootstrapper.kt

```diff
@@ -320,8 +328,18 @@ internal class ModuleInitBootstrapper(
                     }
 
                     postInit(NativeModule::class) {
+                        val ndkService = nativeModule.ndkService
+                        val initWorkerTaskQueueTime = initModule.clock.now()
+                        workerThreadModule.backgroundWorker(WorkerName.SERVICE_INIT).submit {
```

Comment:
Schedule this to run right after the NDK service initializes, hoping to approximate how a background task on this thread will be scheduled in prod. Looking at the difference between the queue and run times should tell us what to expect